src/process_utility.c

```diff
@@ -1688,6 +1687,34 @@ process_altertable_end_subcmd(Hypertable *ht, Node *parsetree, ObjectAddress *ob
 		case AT_DropCluster:
 			foreach_chunk(ht, process_altertable_chunk, cmd);
 			break;
+		case AT_SetTableSpace:
+			{
+				Oid			tspc_oid = get_rel_tablespace(ht->main_table_relid);
```

Comment:
The code handling this command should be moved to a function, like done in the other commands (e.g., `process_altertable_set_tablespace_end`).